python/tvm/contrib/gemmini/tutorials/single_operators/add-tutorial.ipynb

```diff
@@ -0,0 +1,395 @@
+{
```

Comment:
tutorial files should move to somewhere under `gallery/how_to/`. Also you need to change the format to .py file and write it in sphinx format. Now we support notebook generation and google colab, so you can even add cells to install all the dependencies and run it in google colab

Reply:
Thanks for the feedback, I am on it